src/main/java/racingCar/domain/CarRace.java

```diff
@@ -0,0 +1,35 @@
+package racingCar.domain;
+
+import java.util.List;
+
+public class CarRace {
+    private final Cars cars;
+    private final RandomNumber random;
+
+    public CarRace(String[] names) {
+        this(new RandomNumber(), new Cars(names));
+    }
+
+    public CarRace(RandomNumber random, Cars cars) {
+        this.random = random;
+        this.cars = cars;
+    }
+
+    public List<Car> getCars() {
+        return this.cars.value();
+    }
+
+    public void play() {
+        for (Car car : this.cars.value()) {
+            car.move(getRandomNum());
+        }
```

Comment:
```suggestion
        cars.play();
```
일급 콜렉션으로 포장한 후 다시 값을 꺼내고 있음
위와 같이 포장한 객체로 메시지를 보내면 어떨까?